extensions/2.0/Vendor/EXT_feature_metadata/schema/texture.EXT_feature_metadata.features.properties.schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Property texture channels",
+  "type": "object",
+  "description": "Texture channels containing feature data.",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+      "channels": {
+          "type": "string",
+          "pattern": "^(?=[rgba]{1,4}$)(?!.*(.).*\\1).*$",
```

Comment:
I talked about this once with @lilleyse the other day, wondering if it's at all useful to just allow swizzling. Even though it's probably not super useful, it will likely simplify the spec and implementations.

Reply:
I can't think of a good reason to disallow swizzling, so let's go ahead and simplify this.